packages/flet_geolocator/pubspec.yaml

```
@@ -0,0 +1,25 @@
+name: flet_geolocator
+description: Flet Geolocator control
+homepage: https://flet.dev
+repository: https://github.com/flet-dev/flet/packages/flet_geolocator
+version: 0.22.0
```

```suggestion
version: 0.23.0
```

Your addition will most probably be part of Flet v0.23.0, so let's modify.

---

done